vbf_ntuples.py

```diff
@@ -417,7 +443,8 @@ def hlt_accept(path):
             #self.ele_id[idx]  = goodelectrons[idx].mvaIso_Fall17V2_WP80
             self.ele_id[idx]  = goodelectrons[idx].mvaIso
 
-        goodtaus = [t for t in taus if t.pt>20. and abs(t.eta)<2.4 and t.idDeepTau2018v2p5VSe>=4 and t.idDeepTau2018v2p5VSmu>=8 and t.idDeepTau2018v2p5VSjet>=16 ] 
+        # Tau reccomendation for MuTau FS is VVLoose vs Electron, Tight vs Muon, Medium vs Jet
+        goodtaus = [t for t in taus if t.pt>20. and abs(t.eta)<2.4 and t.idDeepTau2018v2p5VSe>=2 and t.idDeepTau2018v2p5VSmu>=4 and t.idDeepTau2018v2p5VSjet>=5 ] 
```

Comment:
Gotcha, I updated the PR so that no idea is applied, but some options are commented out next to it. I also added branches for DeepTau ID. 

Without any ID I would worry that the samples would get too large from carrying around junk taus that don't pass the loosest ID requirements, but maybe that's not an issue. Right now, processing CDE with the ID in the comment above (2, 4, 5), results in 30GB of files from the combined Muon datasets, which is pretty manageable